backend/src/main/resources/schema.graphqls

```diff
@@ -135,6 +136,7 @@ type Query {
   patients(facilityId: String): [Patient]
   patient(id: String!): Patient
   organization: Organization
+  organizations: [Organization]
```

Comment:
It's not a crazy thing to have exist, but I don't actually see it in the spec—what's the use case for this?

Reply:
in order to add a user to an org, we must first know what orgs are out there. this assumes again that the "addUserToOrg" call can add a user to *any* org, not just the current caller's own org.